python/tweets/searcher.py

```diff
@@ -0,0 +1,97 @@
+# This is a "tweet grabber" script - when run, it searches the Twitter archive and
+# finds all tweets matching a certain query, and then stores them in a mongoDB mLab database.
+
+
+# Import modules (you may need to `pip install` these at the command line):
+# - tweepy, for interacting with the Twitter API
+# - json, for working with the tweets themselves (they are returned as json objects)
+# - pymongo, for connecting to a mongoDB mLab database where we'll store the tweets
+import tweepy, json, pymongo
+
+
+# Set up API - replace keys with your own: https://apps.twitter.com/
+CONSUMER_KEY = "<CONSUMER_KEY>"
+CONSUMER_SECRET = "<CONSUMER_SECRET>"
+ACCESS_KEY = "<ACCESS_KEY>"
+ACCESS_SECRET = "<ACCESS_TOKEN>"
+auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+auth.set_access_token(ACCESS_KEY, ACCESS_SECRET)
+api = tweepy.API(auth)
+
+
+# Bath, England place id for twitter api is "1db4f0a70fc5c9db", found using api.geo_search:
+# api.geo_search(query="Bath, England", granularity="city")[0]
+# We'll search for tweets from here
+search_query = "place:1db4f0a70fc5c9db"
+
+# Max tweets to grab in one go (just a big number)
+max_tweets = 1000000000
+# Set to max per query that API allows (100)
+tweets_per_query = 100
+
+# Connect to the mLab database
+client = pymongo.MongoClient("mongodb://shareduser:shareduser@ds123614.mlab.com:23614/bmlm-parking-tweets")
```

Comment:
I take it you're happy sharing these credentials then, as they are now public for the world to see! HMU on Slack if this was unintentional, and we can work through how to undo it.

Reply:
Yep, this one is fine! I set it up for this.